### PR TITLE
Fix sampling for real/imag & nicer plotting

### PR DESCRIPTION
**Pull request by FeGeyer**

### CODE REVIEW

**Comment by FeGeyer** [comment]
Added normalization to `lr_find`

File: docs/changes/152.bigfix.rst
Comment (Kevin2) [inline docs/changes/152.bigfix.rst:1]: typo in name

File: radionets/evaluation/plotting.py [hunk 1]
```diff
@@ -734,17 +799,23 @@ def histogram_peak_intensity(ratios_peak, out_path, plot_format="png"):
         alpha=0.75,
     )
     ax1.axvline(1, color="red", linestyle="dashed")
-    ax1.set_xlabel("Ratio of intensity peak flux")
+    ax1.set_xlabel("Ratio of intensity peak fluxes")
```
Comment (Kevin2) [inline radionets/evaluation/plotting.py:802]: I think this should be flux densities as our source images are in units of Jy/pixel, which is a flux density

Comment (FeGeyer) [reply radionets/evaluation/plotting.py]: Done

**Review by Kevin2 — APPROVED** [review]

**Pull request merged**